listable/static/listable/js/listable.js

```diff
@@ -50,6 +50,59 @@ function listable(moment) {
         iFilteringDelay: Listable.filteringDelay
     });
 
+    if (Listable.liveFilters) {
+        table.fnSettings().aoDrawCallback.push({
+            fn(settings) {
+                // There is no easy way to get the JSON data that was returned from the server last
+                // and no callback that is triggered only on ajax requests, so instead we look at the
+                // jqXHR object that is stored in the settings object to determine the last set of
+                // live filters that were returned from the server.
+                if (!(settings.jqXHR && settings.jqXHR.responseJSON && settings.jqXHR.responseJSON.liveFilters)) {
```

Comment:
Not sure what's going on but `settings.jqXHR.responseJSON` always seems to be undefined for me.

Reply:
Updated the bundled version of jquery. Looks like responseText was being set but not responseJSON in the previous version, even though dataType was correctly set to JSON.

Reply:
Thanks, it's working now!